tests/testthat/test-events.R

```diff
@@ -0,0 +1,236 @@
+test_that("create_event_based_processes assigns a listener to each event", {
+
+  pop <- get_population("Antigua and Barbuda")
+  contact_matrix_set = squire::get_mixing_matrix(iso3c = "ATG")
+  parameters <- get_parameters(pop, 1, 100, contact_matrix_set)
```

Comment:
I suspect you're on it already but this is failing for me:

```
Error in parse_country_severity(country = country, prob_hosp = prob_hosp,  : 
  Country not found
In addition: Warning message:
In if (!country %in% unique(squire::population$country)) { :
  the condition has length > 1 and only the first element will be used
> traceback()
4: stop("Country not found")
3: parse_country_severity(country = country, prob_hosp = prob_hosp, 
       prob_severe = prob_severe, prob_non_severe_death_treatment = prob_non_severe_death_treatment, 
       prob_severe_death_treatment = prob_severe_death_treatment, 
       prob_non_severe_death_no_treatment = prob_non_severe_death_no_treatment, 
       prob_severe_death_no_treatment = prob_severe_death_no_treatment, 
       walker_params = walker_params)
2: squire::parameters_explicit_SEEIR(population = pop$n, dt = dt, 
       time_period = time_period, ...) at parameters.R#11
1: get_parameters(pop, 1, 100, contact_matrix_set)
```

The error (and warning) seem to come from squire though, but it looks like that is because of an error in how get_parameters is calling the squire function?